apier/v1/resourcesv1_it_test.go

```diff
@@ -221,7 +221,7 @@ func testV1RsTTL0(t *testing.T) {
 		Units:   2,
 	}
 	if err := rlsV1Rpc.Call(utils.ResourceSv1AllocateResource, argsRU, &reply); err == nil ||
-		err.Error() != utils.ErrResourceUnavailable.Error() {
+		err.Error() != utils.ErrResourceUnalocated.Error() {
```

Comment:
sorry, my bad here. Please revert to ErrResourceUnavailable since it looks more appropriate as answer